testing/python/jit/test_tilelang_jit_cutedsl.py

```
@@ -311,6 +359,51 @@ def test_cutedsl_dynamic_shape():
+        mbars,
+        trans_A,
+        trans_B,
+        in_dtype,
+        out_dtype,
+        accum_dtype,
+        num_stages,
+        threads,
+    )
+    matmul_kernel = tilelang.compile(program, target="cutedsl")
+
+    assert f"barriers = tl.alloc_smem(cutlass.Uint64, size_in_elems={len(mbars)})" in matmul_kernel.get_kernel_source()  # noqa: E501
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)